src/devtools/client/debugger/src/utils/tabs.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface Tab {
+  url: string;
+  sourceId: string | null;
+}
+
+export function persistTabs(tabs: Tab[]): Tab[];
```

Comment:
`src/devtools/client/debugger/src/utils/tabs.js` is a very simple JS file, we can just move it to TS.